process/release.md

```diff
@@ -31,7 +31,11 @@ There are two significant and different release artifacts for Zowe.  The source
 
 The project officially release source code which can be built into an executable version of Zowe.  This is the core deliverable of Zowe and is intended for downstream consumers that may use Zowe in their projects or products as well as other developers.
 
-The "Convenience Build" is a courtesy release artifact that includes an installer and all available artifacts to run and use Zowe including sample applications.  The convenience build is intended for consumers that simply want to use Zowe and its APIs.  This is built from the source code release above.  Code from other projects that have compatible open source licenses may be included to support overall Zowe capability.  Like bash, git, python as well as zOS-Workflows or other related projects to facilitate a complete software stack for Zowe.
+The "Convenience Build" is composed of two courtesy release artifacts:
+* An installer and all available artifacts to run and use Zowe on z/OS including sample applications.
+* An offline installable version of the Zowe CLI.
+
+The convenience build is intended for consumers that simply want to use Zowe and its APIs.  This is built from the source code release above. Code from other projects that have compatible open source licenses may be included to support overall Zowe capability.  Like bash, git, python as well as zOS-Workflows or other related projects to facilitate a complete software stack for Zowe. Zowe may release its convenience build artifacts at the same time or independently of one another per project discretion and vote in accordance with the normal [release process](#release-approval-procedure).
```

Comment:
> Zowe may release its convenience build artifacts at the same time or independently of one another per project discretion and vote in accordance with the normal [release process](#release-approval-procedure).

Can you clarify this with an example? Not sure if I'm following what the intention and plan is.

Reply:
The primary goal of this change is to have optionally independent delivery cycles for the 2 artifacts below (screenshot from zowe.org): 

![existing_downloads](https://user-images.githubusercontent.com/35308966/65433886-dc06d880-ddeb-11e9-8d75-428daeb78e13.png)

While we normally release these at the same time (1.4.0 shown), we may want to release a new version of the CLI if we find some issue - and set the delivery to 1.4.1, or 1.5.0 if necessary - without incrementing or re-delivering the z/OS components.

The goal is also to extend this independence for "Active Development / Alpha" deliverables, so we can release a Zowe CLI Active Development version on zowe.org, while Zowe z/OS does not yet have a matching artifact version.

Reply:
Got it - definitely understand the motivation.

My gut is that framework releases should stick to a major.minor version numbering, and that separate z/OS and CLI components would then use a major.minor.patch numbering.

For example:

Zowe 1.5 framework release initially would include Zowe CLI 1.5.0 and Zowe z/OS Components 1.5.0.
Then if a patch is needed to the CLI only, you could name that Zowe CLI 1.5.1.

Is that capture the use case?

For Alphas and pre-releases, that's makes sense provided we split them out separately on the website and that we go through the same signoff process for each one built. Alternatively, you might want to direct people to build from source ( not sure where those instructions are ) if they truly want to track nightlies.

Let me know if what I'm saying makes sense.

Reply:
Everything you said makes sense. The version syntax is something I think we should talk through again as we think about Zowe 2.x; we wanted to follow semver for all the Zowe deliverables but in hindsight it can easily lead to a separation / proliferation of minor versions thats more confusing than informational at a glance. We may want to keep components using semver and separate Zowe convenience build versioning from that format.

For Alphas - the only addition I have is we're looking to do the signoff for stable checkpoints on these development lines; nightlies are available from source and will be accessible from Zowe's artifactory server (publicly accessible and more geared towards developers/extenders). I would expect something along the line of monthly updates for Alpha convenience deliverables.

Reply:
@jmertic Build instructions should be included in the projects.  For instance, zLux includes fairly detailed instructions the entail building zLux and related repos.  I don't think there is necessariy a "build everything" apart from the Jenkins pipeline.